include/nuttx/tls.h

```diff
@@ -205,6 +205,12 @@ struct tls_info_s
   struct pthread_cleanup_s stack[CONFIG_PTHREAD_CLEANUP_STACKSIZE];
```

Comment:
could you create a new patch adding tl_ prefix to tos and stack

Reply:
Ok